op25/gr-op25_repeater/apps/multi_rx.py

```diff
@@ -581,6 +582,7 @@ def __init__(self, verbosity, config):
         self.ui_out_q = gr.msg_queue(100)
         self.ui_timeout = 5.0
         self.ui_last_update = 0.0
+        self.http_plot_type = 'image'
```

Comment:
It might make more sense to just call `self.http_plot_type` just `self.plot_type` as it is used in areas where `http` might not be the terminal type, and it does default to `image`.